pkg/controllers/scheduler/scheduler.go

```diff
@@ -339,16 +341,41 @@ func (s *Scheduler) reconcile(qualifiedName common.QualifiedName) (status worker
 			)
 			return worker.StatusError
 		}
-		profile, err := s.profileForFedObject(fedObject, s.buildFrameworkHandle())
+
+		var schedulingProfile *fedcorev1a1.SchedulingProfile
+		profileName := policy.GetSpec().SchedulingProfile
+		if len(profileName) > 0 {
+			schedulingProfile, err = s.schedulingProfileLister.Get(profileName)
+			if err != nil {
+				// TODO(hawjia): handle error - should we retry if not found?
+				keyedLogger.Error(err, "Failed to get scheduling profile")
```

Comment:
```suggestion
				keyedLogger.Error(err, "Failed to get scheduling profile", "profile", profileName)
```

Reply:
I'm gonna update `keyedLogger` at the start of this block with `["profile", profileName]` kv

Reply:
fixed